docs: clarify that Desktop refers to macOS only

MrTadeu opened the pull request:
- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### 📋 Changes

Updated the documentation to specify **macOS** instead of "Desktop" to clarify platform support for `auth0_flutter`.

This change is important to prevent developer confusion and to set the correct expectations regarding platform support, as highlighted in the related issue.

### 📎 References

This pull request addresses and fixes the documentation issue discussed in #655.

### 🎯 Testing

This is a documentation-only update and does not require any code-level testing. Reviewers can verify the change by inspecting the diff of the updated documentation file(s).

pmathew92 commented:
@MrTadeu ,  Thank you for raising this PR. For us to merge this PR, could you please sign your commits and update the same ?

MrTadeu commented:
Hi @pmathew92 , the pull request has been updated. The commit have now been signed.

pmathew92 commented on auth0_flutter/example/.env.example:
```diff
@@ -14,4 +14,4 @@ AUTH0_CLIENT_ID=YOUR_AUTH0_CLIENT_ID
 # 2. Update the scheme value in android/app/src/main/res/values/strings.xml
 #
-AUTH0_CUSTOM_SCHEME=YOUR_AUTH0_CUSTOM_SCHEME
+AUTH0_CUSTOM_SCHEME=com.neo.flow.bi://desktop/callback
```
Please revert to the default value

pmathew92 commented on auth0_flutter/example/.env.example:
```diff
@@ -1,11 +1,11 @@
 # The Client Id of your Auth0 application.
 #
-AUTH0_CLIENT_ID=YOUR_AUTH0_CLIENT_ID
+AUTH0_CLIENT_ID=d6FvYRRSKEbob5zbNhY0pQ8KGVA8ZbFA
```
Please revert to the default value

pmathew92 commented on auth0_flutter/example/.env.example:
```diff
@@ -1,11 +1,11 @@
 # Your Auth0 Domain.
 #
-AUTH0_DOMAIN=YOUR_AUTH0_DOMAIN
+AUTH0_DOMAIN=neoflow-bi.eu.auth0.com
```
Please revert to the default value

pmathew92 commented on auth0_flutter/example/lib/example_app.dart:
```diff
@@ -106,11 +106,13 @@ class _ExampleAppState extends State<ExampleApp> {
     // Platform messages may fail, so we use a try/catch PlatformException.
     // We also handle the message potentially returning null.
     try {
+      print("\x1B[32mAPI Login starting...\x1B[0m");
```
Please remove this

pmathew92 commented on auth0_flutter/example/lib/example_app.dart:
```diff
@@ -106,11 +106,13 @@ class _ExampleAppState extends State<ExampleApp> {
           usernameOrEmail: usernameOrEmail,
           password: password,
-          connectionOrRealm: 'Username-Password-Authentication');
+          connectionOrRealm: 'azure-ad-auth0-NeoFlowBi');
```
Please revert to the default value

pmathew92 commented on auth0_flutter/example/windows/flutter/CMakeLists.txt:
```diff
@@ -10,6 +10,11 @@ include(${EPHEMERAL_DIR}/generated_config.cmake)
 # https://github.com/flutter/flutter/issues/57146.
```
The entire windows folder is not needed

pmathew92 reviewed (changes requested):
Hi @MrTadeu , Looks like you have accidentally hardcoded you tenant and user details in the sample apps. Please revert them to the original values and keep only the doc related changes. Also there are changes added in some other files in the windows folder. Those are also not needed as part of this PR . Thank you for quickly fixing the docs

MrTadeu commented:
Hi @pmathew92,

Thank you for your review. I’ve reverted all hardcoded values and removed the unnecessary changes, so the branch now contains only the intended documentation updates.
Apologies for the earlier confusion and any inconvenience caused. I appreciate your guidance in keeping the PR clean.
